packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -31,22 +31,39 @@ @interface FLTImagePickerPlugin () <UINavigationControllerDelegate,
                                     PHPickerViewControllerDelegate,
                                     UIAdaptivePresentationControllerDelegate>
 
+/**
+ * Gets or sets the maximum amount of images that are allowed to be picked.
```

Comment:
Minor nit: for all of these you can document them as if they were variables rather than methods; that's the normal style for properties. So this would be "The maximum number of images that are allowed to be picked." and similarly for the others below.

Reply:
Fixed.